### PR TITLE
Enable Encryption in Modules example fragment

### PR DESCRIPTION
**Pull request by MalikAtalla-AWS**
Enable Encryption in Modules example fragment, because that's good practice in S3

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: fragments/sample.json [hunk 1]
```diff
@@ -15,6 +15,15 @@
...
                 },
+                "BucketEncryption": {
+                    "ServerSideEncryptionConfiguration": [
+                        {
+                            "ServerSideEncryptionByDefault": {
+                                "SSEAlgorithm": "AES256"
+                            }
+                        }
+                    ]
+                },
```
Comment (MalikAtalla-AWS) [inline fragments/sample.json:26]: This whole file really doesn't need to be here. It's an artifact of the modules unit tests. I wasn't able to figure out how to get rid of it today with my time constraints, but will do this in a separate PR.

**Review by PatMyron — APPROVED** [review]

**Review by miparnisari — APPROVED** [review]
